lib/Inst/Inst.cpp

```diff
@@ -584,6 +590,13 @@ Inst *InstContext::createVar(unsigned Width, llvm::StringRef Name,
   return I;
 }
 
+Inst *InstContext::createVar(unsigned Width, llvm::StringRef Name) {
+  return createVar(Width, Name, llvm::ConstantRange(Width, /*isFullSet*/ true),
+                   /*KnownZero*/ llvm::APInt(Width, 0), /*KnownOnes*/ llvm::APInt(Width, 0),
+                   /*NonZero*/ false, /*NonNegative*/ false, /*PowerOfTwo*/ false,
+                   /*Negative*/ false, /*SignBits*/ 1);
```

Comment:
for all of these please follow the existing idiom, for example:
/*Negative=*/false